tree/tree/src/TLeafD32.cxx

```diff
@@ -204,8 +208,10 @@ void TLeafD32::Streamer(TBuffer &R__b)
    if (R__b.IsReading()) {
       R__b.ReadClassBuffer(TLeafD32::Class(), this);
 
-      if (fElement && TString(fElement->GetTitle()).Contains("["))
-         fElement = new TStreamerElement(Form("%s_Element", fName.Data()), fElement->GetTitle(), 0, 0, "Double32_t");
+      if (fTitle.Contains("[")) {
+         auto slash = fTitle.First("/");
```

Comment:
Is the behavior correct if there is no slashes?

Reply:
Yes, I had checked that. It's -1 so it works for both cases

Reply:
Fair enough.  I would make the code clear (and more resilient in case that value changes) by test of it (`TString::kNPOS`)